stcgal/protocols.py

```diff
@@ -1653,6 +1693,7 @@ def program_options(self):
 
         print("Target UID: %s" % Utils.hexstr(self.uid))
 
+
```

Comment:
Is this white space change needed? I don't mind whitespace changes if they make formatting cleaner or more consistent, but please don't mix them unnecessarily with commits that change functionality. It looks like this is just added by accident, though.